### ISSUE TITLE
phenix real space refine issue

### ISSUE THREAD

**Issue by JuhaHuiskonen** [scipion-em/scipion-em-phenix#7]
If I run 'phenix - realspace' refine from Scipion GUI (on a model that I have fitted in place with Chimera and imported to Scipion), it prints model-to-map fit CC_mask value 0.000 in the beginning (the rest of the stats don't make much sense either).   Running real space refinement from Phenix GUI (the same binaries) shows CC_mask value of 0.80, using the same map and model (and the rest of the stats are fine). Any idea why?

**Comment by rmarabini** [comment]
Hi,

 What version of phenix and scipion-em are you using?

**Comment by vabrishami** [comment]
Hi @rmarabini,

The plugin version is: 1.0.5
The Phenix version is: 1.16-3549

**Comment by rmarabini** [comment]
Hi Vahid,

  Sorry for the long delay answering your question. A small problem that we
had with model_building stuff
is that it has been developed for the developer version of scipion
rather than the stable one. During this last month we have introduced some
small modifications in order to make the code fully compatible with
scpion 2.0.

   In summary the, best way to process would be to uninstall
scipion-em-phenix (and scipion-em-chimera, scipion-em-atomstructutils,
scipion-em-ccp4 and scipion-em-powerfit) and install it again using the
menu in the main scipion window (Configuration -> Plugin). Let me know if
you have any further problem. (If you want to install from the github
repository use the master branch since the devel branch will only work with
the next scipion 3 version)

I have a question for you about  opic-oxford/relion-relax. Is there any
document in which it is described in detail how this modification of the
relion package works. In particulas from all symmetry related particles
does the program  keep one of them in the final recosntruction?

        Thanks for the help

   Roberto

**Comment by vabrishami** [comment]
Hi Roberto,

Thanks for the help. I will try to check it and let you know if it works or not. Actually we have a cleaner implementation (and faster than previous one) in Relion 3.0, and you can find it here:

https://bitbucket.org/LSB-Helsinki/relion-3.0/src/master/

Yes, actually this is the idea. So we search a range (depends on --sigma_ang) around symmetry mates (multiple priors) of a particle and try to choose the best fit. However, orientations related to the symmetry mates of a particle normally aren't in the healpix library, and so it chooses a healpix value which is close to real value. So remember to set a high healpix order in Scipion to have the closest possible healpix value.

In Summary:

1. Add --relax_sym (for example C5) to additional parameters.
2. Choose a high healpix order.
3. Add --sgima_ang option to additional parameters. If you want to be very restrict, set it to a very small value (0.001 maybe) so that it just checks on orientation.

**Comment by rmarabini** [comment]
Hi,

  Then,  --relax_sym  is a new parameter of relion_refine, isn't it?

   Roberto

**Comment by vabrishami** [comment]
Yes. --relax_sym [symmetry_group].

**Comment by rmarabini** [comment]
I think this is fixed. If this is not the case pleae complain.
